lib/locale/pl/strings.php

```diff
@@ -270,6 +271,7 @@
 $_LANG['Change connection type'] = 'Zmień typ połączenia';
 $_LANG['Connection type:'] = 'Typ połączenia:';
 $_LANG['incoming'] = 'przychodzące';
+$_LANG['Allows you to print IP addresses from whole network.'] = 'Umożliwia drukowanie adresów IP z całej sieci.';
```

Comment:
Po angielsku:
Allows you to display IP address usage from whole network.
Po polsku:
Umożliwia wyświetlenie użycia adresów IP z całej sieci.